app/models/oral_history_content.rb

```diff
@@ -105,6 +118,52 @@ def combined_audio_derivatives_job_status=(value)
     self.combined_audio_derivatives_job_status_changed_at = DateTime.now
   end
 
+  def interviewee_birth_place
+    @interviewee_birth_place ||= begin
+      return nil if interviewee_birth.nil?
+      interviewee_birth.to_h['place']
+    end
+  end
```

Comment:
OK, you don't need #to_h here to convert it to a hash and then access it as a hash key -- it's a model object that already has a `place` method. So the right implementation is just `interviewee_birth.place`

If you want to guard against interviewee_birth being nil, you can just use the ruby "safe navigation operator", so:

    def inteviewee_birth_place
       interviewee_birth&.place
    end

There is no need to memoize it, as it's not at all expensive to look up. 

At that point, I think there is no need for these wrapper methods. Code that wants this can just call `oral_history_content&.interviewee_birth&.place` directly. 

Or if you disagree and want the wrapper method I could be convinced, but it should be this simple one-liner. Same for all following similar methods.

Reply:
Removed the memoizers -- keeping the wrapper methods, though.